src/components/menu/Menu.component.tsx

```diff
@@ -1,11 +1,65 @@
-import React from 'react';
+import React from "react";
+import img1 from './images/menu1.png'; 
+import header from './images/header.png';
+import img2 from './images/menu2.png';
+import style from './Menu.module.css';
 import Footer from '../footer/Footer';
 
-export default function Menu() {
+
+ function Menu() {
+   const image1 = "https://www.notion.so/janac/Menu-Page-dishes-c97e397570384b69a4ce291d02dde758#88140f350d3e40d5986c2a7791ae3709";
   return (
-    <div>
-        You like snacks?Yes!
-        <Footer/>
+
+  <body>
+
+  <img src = {header}
+   style = {{
```

Comment:
Move these inline styles to the module CSS file - ideally we want to avoid inline styles whenever possible. You can target these specific elements by adding a class here and referencing the same class in your css module file.